### ISSUE TITLE
training dataset

### ISSUE THREAD

**Issue by yenchoupan** [RookieJunChen/Inter-SubNet#6]
I'm trying to retrain your InterSubnet. I have some questions about training datasets.
1. I saw you used interspeech 2020 datasets in your code but the subset of the interspeech 2021 datasets in your paper. Which is correct ?
2. If you use the subset of interspeech 2021 datasets, what kind of dataset did you use? Fullband or Wideband? Only use clean read_speech or do you use emotional speech and non-English speech?
Thanks

**Comment by RookieJunChen** [comment]
Our training dataset mainly consists of the wideband (16kHz) data from the DNS Challenge at InterSpeech 2021. The clean dataset used is "readbook" and the noise dataset incorporates the complete set of noises. Our test dataset utilizes the referenced test set released in the DNS Challenge at InterSpeech 2020.

**Comment by yenchoupan** [comment]
Did you use drop band  that reaches the performance in your paper?

**Comment by RookieJunChen** [comment]
The configs of my final reported results are listed in [config.](https://github.com/RookieJunChen/Inter-SubNet/tree/master/config/train.toml)
If there is a slight difference in your results, I think it's likely to be a randomness difference due to the dynamic mixing strategy. Besides, the type of GPU also has some effect on the results.
